### DISCUSSION TITLE
Firebird 4 for Mac

### DISCUSSION THREAD

**Discussion by germanobender** [FirebirdSQL/firebird#7746] [General]
Hi!
I'm looking for the firebird 4 installer for **MACOS**!
Where do I find?

**Comment by machacekmartin** [comment]
Have you found any solution on this?

**Reply by mrotteveel** [reply]
There isn't any. Use Firebird 5 instead.

**Reply by aafemt** [reply]
If they have enough knowledge in Mac, they can build it from sources.
